contracts/Governance/GovernorBravoDelegateG1.sol

```diff
@@ -9,25 +9,25 @@ contract GovernorBravoDelegate is GovernorBravoDelegateStorageV1, GovernorBravoE
     string public constant name = "Mobius Governance";
 
     /// @notice The minimum setable proposal threshold
-    uint public constant MIN_PROPOSAL_THRESHOLD = 1000000e18; // 1,000,000 veMOBI
+    uint public constant MIN_PROPOSAL_THRESHOLD = 500e18; // 500 veMOBI
 
     /// @notice The maximum setable proposal threshold
-    uint public constant MAX_PROPOSAL_THRESHOLD =  20000000e18; //20,000,000 veMOBI
+    uint public constant MAX_PROPOSAL_THRESHOLD =  10000e18; //10,000 veMOBI
 
     /// @notice The minimum setable voting period
-    uint public constant MIN_VOTING_PERIOD = 20; // About 100 seconds
+    uint public constant MIN_VOTING_PERIOD =  17280; // 17280 blocks, about 1 day
 
     /// @notice The max setable voting period
-    uint public constant MAX_VOTING_PERIOD = 120960; // About 1 week
+    uint public constant MAX_VOTING_PERIOD = 120960; // 120960 blocks, about 1 week
 
     /// @notice The min setable voting delay
     uint public constant MIN_VOTING_DELAY = 1;
 
     /// @notice The max setable voting delay
-    uint public constant MAX_VOTING_DELAY = 120960; // About 1 week
+    uint public constant MAX_VOTING_DELAY = 69120; // 69120 blocks, about 4 days
 
     /// @notice The number of votes in support of a proposal required in order for a quorum to be reached and for a vote to succeed
-    uint public constant quorumVotes =  20000000e18; // 400,000 = 4% of veMOBI
+    uint public constant quorumVotes =  4000000e18; // 4,000,000 veMOBI
```

Comment:
im not a solidity dev but is  it sounds like constant means unchangable forever. If veMobi circulation is increasing id think we would want quorum to be able to increase too

Reply:
Yes you couldn't change this number directly, but since the GovernorBravoDelegator is a proxy, you could effectively overwrite anything in the GovernorBravoDelegateG1 file

Reply:
I think the thing thats weird to me is that the other parameters that are labled "start" in the PR description are set in the init but this one is labeled "start" but set like the MAX and MINs

Reply:
That's fair, bad naming convention on my part